package.json

```diff
@@ -24,7 +24,7 @@
   "homepage": "https://github.com/tin0312/walk-more-than-you-scroll#readme",
   "devDependencies": {
     "@types/chrome": "^0.0.242",
-    "@types/jquery": "^3.5.16",
+    "@types/jquery": "^3.5.18",
```

Comment:
Why?

Reply:
I accidently updated it by:

 npm i --save-dev @types/jquery